Account/FxAPushMessageHandler.swift

```diff
@@ -62,64 +62,62 @@ extension FxAPushMessageHandler {
 
         // return handle(plaintext: string)
         let deferred = PushMessageResult()
-        RustFirefoxAccounts.startup(prefs: profile.prefs).uponQueue(.main) { _ in
-            RustFirefoxAccounts.reconfig { accountManager in
-                accountManager.deviceConstellation()?.processRawIncomingAccountEvent(pushPayload: string) {
-                    result in
-                    guard case .success(let events) = result, let firstEvent = events.first else {
-                        let err: PushMessageError
-                        if case .failure(let error) = result {
-                            err = PushMessageError.messageIncomplete(error.localizedDescription)
-                        } else {
-                            err = PushMessageError.messageIncomplete("empty message")
-                        }
-                        deferred.fill(Maybe(failure: err))
-                        return
+        RustFirefoxAccounts.reconfig(prefs: profile.prefs).uponQueue(.main) { accountManager in
+            accountManager.deviceConstellation()?.processRawIncomingAccountEvent(pushPayload: string) {
```

Comment:
This is actually a 2-line changing which changes the indentation, review with ignore whitespace changes.